src/pages/ReplaceServer/ReplaceController.js

```diff
@@ -371,8 +371,7 @@ class ReplaceController extends BaseUpdateWizardPage {
   }
 
   render() {
-    //if error happens, cancel button shows up
-    let cancel =  this.state.overallStatus === constants.STATUS.FAILED;
+    let failed = (this.state.overallStatus === STATUS.FAILED);
```

Comment:
you need to keep the consttnts. otherwise it will have undefined error and lint error as well

Reply:
Good catch. I missed this when resolving merge conflicts.